src/Views/scstormrunoff.html

```diff
@@ -378,12 +378,290 @@ <h4 class="modal-title">New {{flowType.displayName}} Segment</h4>
+                                        <input type ="number" name="slopeSteepness" ng-model="vm.slopeSteepness" required></input>
+                                        <label for="coverName">Cover Name<span style="color: #ffcc00">*</span></label>
+                                        <select name="coverName" ng-model="vm.coverName" ng-options="c for c in vm.coverNames" required></select>     
+                                        <label for="practiceName">Practice Name<span style="color: #ffcc00">*</span></label>
+                                        <select name="practiceName" ng-model="vm.practiceName" ng-options="p for p in vm.practiceNames" required></select>                                                                      
+                                    </div>
+                                    <button style="line-height:unset;" class="btn-wim-sm btn-wim-success btn-wim-group-1 btn-blue" ng-if="vm.usle" ng-click="vm.usle = !vm.usle">Remove USLE & MUSLE</button>
+                                </div>
+
+                                <!-- Storm Ponding -->
+                                <div>
+                                    <button style="line-height:unset;" class="btn-wim-sm btn-wim-success btn-wim-group-1 btn-blue" ng-if="!vm.stormponds" ng-click="vm.stormponds = !vm.stormponds">Design Storm Ponds</button>
+                                    <div ng-if="vm.stormponds">
+                                        <h3 class="center-div">Design Storm Ponds</h3>
+                                        <div class="wim-alert hidden-print">Select a 'Pond Option' and fill out all required fields.</div>
+                                        <label for="pondOption">Pond Option:<span style="color: #ffcc00">*</span></label>
+                                        <select name="pondOption" ng-model="vm.pondOption" ng-change="vm.disableSediment()" required>
+                                            <option value="1">Pond Option 1 (Inverted Quadrilateral Frustum)</option>
+                                            <option value="2">Pond Option 2 (Elevation vs Surface Area)</option>
+                                        </select>
+
+                                        <label for="pond_bottom_elev">Bottom Elevation (feet):<span style="color: #ffcc00">*</span></label><span ng-if="SyntheticUrbanHydrograph.pond_bottom_elev.$error.pattern" class="red label"><span class="fa fa-exclamation-triangle"></span>&nbsp;Value must be greater than 0</span>
+                                        <input type="number" name="pond_bottom_elev" ng-model="vm.pond_bottom_elev" ng-minlength="1" ng-pattern="greaterThanZero" required/>
+
+                                        <label for="max_depth">Max Depth (feet):</label>
+                                        <input type="number" name="max_depth" ng-model="vm.max_depth" readonly/>
```

Comment:
Is there a way to signify that this input field is disabled? Maybe add a disabled attribute? So the user knows this one is not editable.